lib/commands/connect/import.js

```diff
@@ -23,18 +22,10 @@ module.exports = {
           } else {
             let connection = connections[0];
             let url = 'https://' + api.HOST + ':443' + '/api/v3/connections/' + connection.id + '/actions/import';
-            fs.stat(context.args.file, function(err, stats) {
-              rest.post(url, {
-                headers: {
-                  Authorization: 'Bearer ' + context.auth.password
-                },
-                multipart: true,
-                data: {
-                  'config': rest.file(context.args.file, null, stats.size, null, context.args.file)
-                }
-              }).on('complete', function(data) {
+            fs.readFile(context.args.file, 'utf8', function(err, data) {
+              api.request(context.auth.password, 'POST', url, data).then(function(response) {
```

Comment:
I wonder if we should JSON.parse data and not have the jsonify conditional. Provides the benefit of having invalid json detected in the client and keeps a single path through api.request.